wiki/faq.md

```diff
@@ -654,3 +659,25 @@ it can be impossible to send. To fix this, you can try consolidating them using
 try running `consolidateunspent <address>`. After this finishes, try sending
 the transaction again. This may require multiple rounds of the command if you
 have a lot of small transactions.
+
+---
+## Windows Only
+
+---
+
+### Cannot read configuration file. Please check the path and format of the file
+
+First, check you delete the `%appdata%/GridcoinResearch` folder. You should not remove this folder.
+If this folder is not at your `%appdata%`, recreate the folder and add the files `wallet.dat` and `gridcoinresearch.conf` inside the folder.
+
+---
+### How can I move my data folder?
+
+1. Close your wallet.
+2. Move your entire data from actual location to the new location. The default location of the data is `%appdata%\GridcoinResearch`.
+3. After you move the entire contents, right click at the Gridcoin Icon, select properties, and add the `-choosedatadir` to the end.
+4. Run Gridcoin.
+5. It will show the dialog dialog box to choose the data directory. 
+6. Choose the new location.
+

```

Comment:
Probably should mention that you want to remove that `-choosedatadir` flag after you are done. Otherwise I just have some grammatical fixes/simplifications

```suggestion
1. Close your wallet.
2. Move all data from the current location to the new location. The default location of the data is `%appdata%\GridcoinResearch`.
3. After you move the entire contents, right click the Gridcoin Shortcut, select properties, and add `-choosedatadir` to the end.
4. Run the wallet.
5. It will show the dialog box to choose the data directory. 
6. Choose the new location.
7. Right-click the Gridcoin Shortcut, select properties, and remove `-choosedatadir`


```